core/services/ocr2/delegate.go

```diff
@@ -317,34 +315,43 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 	}
 
 	if spec.Relay == relay.EVM {
```

Comment:
Can we have a single if branch that says something in the lines: 
if (forwardingAllowed && relay in supportedRelays && job in supportedJobs) {
// forwarder flow setting sendingKeys
} else {
// default flow setting sendingKeys
}

Reply:
+1 to that but also instead of having multiple nested conditions, have a single condition to either use forwarders or default flow